public/content/developers/docs/nodes-and-clients/index.md

```diff
@@ -234,21 +234,38 @@ Synchronization modes represent different approaches to this process with variou
 
 ### Execution layer sync modes {#execution-layer-sync-modes}
 
-#### Full archive sync {#full-sync}
+#### Full node syncs
 
-Full sync downloads all blocks (including headers, transactions, and receipts) and generates the state of the blockchain incrementally by executing every block from genesis.
+You may run a full node with a:
+
+- [full sync](#full-sync)
+- [fast sync](#fast-sync)
+- [snap sync](#snap-sync)
```

Comment:
I'm not sure having a bulleted list with hash links is necessary here... the content isn't very long, might suggest we just jump right into the sync options.

Also, ideally we keep the content tree structure to no deeper than h4 (`####`)... any way we could adjust this so we get straight into the options under "Execution layer sync modes"? like `#### Full node: full sync`, `#### Full node: fast sync`, `#### Full node: snap sync`?

Reply:
fixed with sentence structure instead

Reply:
fixed